src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsFrom(c As Char, c0 As Char, c1 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1)
```

Comment:
OrElse seems unnecessary here. OR with unconditional extra char compare is likely to be cheaper than deciding whether to do it or not.